cni_stop.sh

```diff
@@ -52,6 +52,51 @@ function delete_chache_files {
     rm -rf /var/lib/cni/networks
 }
 
+
+function reset_vfs_guids {
```

Comment:
is reload the driver will have the same effect?

Reply:
@moshe010 , reloading the driver will set them all to zeros.

Reply:
so you want to clean the guids this is the best way. Or I don't understand you change...

Reply:
ok, it did not occur to me to try reloading the drivers.